viewer/packages/tools/src/Measurement/MeasurementLabel.ts

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { Cognite3DViewer } from '@reveal/core';
+import * as THREE from 'three';
+import { HtmlOverlayTool, HtmlOverlayToolOptions } from '../HtmlOverlay/HtmlOverlayTool';
+import labelCSS from './styles/Label.css';
+
+export class MeasurementLabel {
+  private readonly _htmlOverlay: HtmlOverlayTool;
+  private _labelElement: HTMLDivElement;
+  private readonly _handleClustering = this.createEmptyClusterElement.bind(this);
+  private readonly options: HtmlOverlayToolOptions = {
+    clusteringOptions: { mode: 'overlapInScreenSpace', createClusterElementCallback: this._handleClustering }
+  };
+  private static readonly stylesId = 'reveal-viewer-label';
```

Comment:
Should probably be more specific, e.g. `reveal-measurement-label` (if its not shared with some other usage that is)